src/gprs.c

```diff
@@ -1984,9 +1984,10 @@ static DBusMessage *gprs_set_property(DBusConnection *conn,
 					gprs->settings);
 		}
 
+		notify_powered_change(gprs);
+
 		gprs_netreg_update(gprs);
 
-		notify_powered_change(gprs);
```

Comment:
The changes to add notify_powered_change() were not submitted. It was not clear how to implement dual SIM from upstream's POV. Although I agree on start submitting things as soon as we rebase to ofono 1.18.